### ISSUE TITLE
The isRtlLanguage method returns false for the RTL Kurdish (Sorani).

### ISSUE THREAD

**Issue by nero-angela** [dart-lang/i18n#827]
Hi,

Central Kurdish, also known as [Kurdish (Sorani)](https://en.wikipedia.org/wiki/Sorani), is a right-to-left language.It seems that `isRtlLanguage()` is returning the wrong value as `false`.

<img width="810" alt="image" src="https://github.com/dart-lang/i18n/assets/26322627/0116c335-1137-463e-84a5-79cd0e1232c6">

<details><summary>Code</summary>

```dart
import 'package:intl/intl.dart';

void main() {
  /// Kurdish (Sorani) : https://en.wikipedia.org/wiki/Sorani
  print(Bidi.isRtlLanguage("ckb"));
}
```
</details>

**Comment by mosuem** [comment]
I think the problem is that `ckb` is not a known locale for `package:intl`.

**Comment by badeesAbood** [comment]
@mosuem this issue should be resolved as I mentioned it in PR #863
